src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -180,9 +200,10 @@ internal static string UnderlyingMemberName(int position)
         /// </summary>
         internal static bool IsMemberNameReserved(string name)
         {
-            return memberNames.Contains(name);
+            // PROTOTYPE handle "Rest" and others like "ToString"?
+            return memberNames.Contains(name, StringComparer.Ordinal);
```

Comment:
I think Vlad recently added special ordinal comparer for strings. Should we be using it here instead?